src/main/java/org/frankframework/frankdoc/cmd/XmlAgainstXsdValidator.java

```diff
@@ -0,0 +1,132 @@
+/* 
```

Comment:
Dit moet in een andere module. De 'hoofd' module moet niet de 'test zijn om de xsd te testen'.

Reply:
@nielsm5, je hebt me uitgelegd dat dit een multi-module project moet worden vanwege https://github.com/ibissource/frank-doc/issues/105. We werden het eens dat dit issue het beste opgepakt kan worden voordat we deze class in een aparte Maven module stoppen.